tenseal/cpp/tensors/bfvtensor.h

```diff
@@ -0,0 +1,193 @@
+#ifndef TENSEAL_TENSOR_BFVTENSOR_H
+#define TENSEAL_TENSOR_BFVTENSOR_H
+
+#include "tenseal/cpp/tensors/encrypted_tensor.h"
+#include "tenseal/cpp/tensors/plain_tensor.h"
+#include "tenseal/proto/tensors.pb.h"
+
+namespace tenseal {
+
+using namespace seal;
+using namespace std;
+
+class BFVTensor : public EncryptedTensor<int64_t, shared_ptr<BFVTensor>>,
+                  public enable_shared_from_this<BFVTensor> {
+   public:
+    using EncryptedTensor<int64_t, shared_ptr<BFVTensor>>::decrypt;
+    /**
+     * Create a new BFVTensor from an 1D vector.
```

Comment:
The comment here is not correct, the BFVTensor can be created from tensors of any shape

Reply:
> LGTM! Great work! Thank you for your contribution, @comidan!
> 
> I've left just one minor observation, after that we can merge the PR.

Thank you for the help!

Sure ok, I'll just pushed the new header file with the fixed comment.